### DISCUSSION TITLE
Generated Skeletal Mesh is not rendering on UE5.3

### DISCUSSION THREAD

**Discussion by matiasgql** [AndreaCatania/RuntimeSkeletalMeshGenerator#25] [Q&A]
I have this code to copy an existing skeletal mesh and convert into a Runtime Skeletal Mesh to test if it works and edit that code to edit meshes later.
![image](https://github.com/AndreaCatania/RuntimeSkeletalMeshGenerator/assets/44686929/e731066f-b370-4cb7-9be8-0411360ab57b)
But when its time to render the mesh, just disappear, I was trying to fix it by myself a long time and I can't get it working, it has the vertices count, materials, everything right, but the mesh isn't rendering. Its everything okey with the skeletal mesh component, It works when I use the Skeletal Mesh before creating the Runtime Skeletal Mesh. What can I do?

**Comment by AndreaCatania** [comment]
The plugin is well tested with UE4.27 but not with UE5. That could be the issue. If you find the cause, consider providing a fix via a PR.
